### DISCUSSION TITLE
Welcome to HootingTime Discussions!

### DISCUSSION THREAD

**Discussion by HootHootN64** [HootHootN64/HootingTime#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dopplebockk** [comment]
Long-time emulator and have been playing MK64 since childhood with loose joysticks. I've recently got my set-up optimized with original n64 controllers outfitted with 8bitdo's bluetooth conversion kit. Stumbled back on romhacking.net and noticed the discord link for this project has expired - can a new one be generated if the community is still alive?

**Reply by HootHootN64** [reply]
Hi, this link doesn't work? 
https://discord.gg/ZUQjdMtQyG

**Comment by Todd-3465** [comment]
hey, i tried to host a mk64 hooting time v0.2 game. we wanna use the fullscreen online support. but every time player 2/3/4 joins to player 1 they got an error that p1 is playing a different mk64 version. your z64 files for all players (1 to 4) have the same CRC1 + CRC2 exept P1. P1 has a different CRC2. and that's the reason. and ideas how i can solve the problem? it seems that the z64 patch for P1 ist incorrect? thank u very much

**Reply by HootHootN64** [reply]
Hi, sorry looks like I did some final changes to the ROM back then and forgot to update the other patches. Please try these and confirm if they are working as intended thx: 
~DELETED LINK~

**Reply by Todd-3465** [reply]
hi, every player has his own patched rom as intended. P2, P3, and P4 can join now to P1 as intended. but when i hold D-Up every player see P1 in fullscreen.  this also the case if i try it just with two players. it shows always just P1. :/ any idea?

**Reply by HootHootN64** [reply]
Turns out the P1 patch is more advanced than it should be for that version. Please try with this P1 patch and the original P2-P4 patches (not the new ones I provided earlier): https://github.com/HootHootN64/HootingTime/blob/main/Old_Versions/0.2_HootingTime(P1OnlineFix).zip

Tested it myself in a 2P match and it should work as intended now. Sorry for the inconvenience. Hope you are gonna have a Hooting Time after all.

**Reply by Todd-3465** [reply]
hi, impressive! it works! thank u very much!
Is there a reason why you removed the fullscreen function in later versions?

**Reply by HootHootN64** [reply]
Hooting Time 0.3/1.0 is basically a fresh build to implement 4 player grand prix, so old features were removed completely or not yet integrated again. This exact feature is coming back in version 1.4.

**Reply by Todd-3465** [reply]
can't wait for it :-)
